### PR TITLE
progress on the info page

### PR DESCRIPTION
**Pull request by thirsty-wing**
I need to make some tests for this, but this is the EOD state.

### CODE REVIEW

File: components/BackgroundView/BackgroundView.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  const {
+    children = [],
+  } = props;
+
+  const insets = useSafeAreaInsets();
+
+  return(
+    <View
+      style={{
```
Comment (bc-bane) [inline components/BackgroundView/BackgroundView.jsx:25]: I'd recommend moving this into a react native stylesheet and into a seperate BackgroundView.styles.js file

File: components/ContinentScreen/ContinentScreen.jsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        countries {
+          name
+          code
+          emoji
+        }
+      }
+    }`
+  );
+
+  const isLoading = loading;
```
Comment (bc-bane) [inline components/ContinentScreen/ContinentScreen.jsx]: you could do this on line 15 as 
```
  const [
    data,
    loading: isLoading,
    error,
    refetch,
  ]
```

File: components/ContinentScreen/ContinentScreen.jsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                <Text
+                  style={{
+                    color: theme.palette.text.primary,
+                  }}
+                >
+                  Back
+                </Text>
+              </Pressable>
+              <View
+                style={{
```
Comment (bc-bane) [inline components/ContinentScreen/ContinentScreen.jsx]: pull into stylesheet for this component

File: components/ContinentScreen/ContinentScreen.jsx [hunk 4]
```diff
@@ -0,0 +1,123 @@
+import { ActivityIndicator, ScrollView, Text, View, Pressable } from "react-native";
+import { useLocalSearchParams, useRouter } from "expo-router";
+
+import { theme } from "../../utils/theme";
+import BackgroundView from "../BackgroundView";
+import useFetchEarth from "../../utils/hooks/useFetchEarth";
+
+
+function ContinentScreen() {
+  const params = useLocalSearchParams();
```
Comment (bc-bane) [inline components/ContinentScreen/ContinentScreen.jsx]: move these hooks and api calls out into a separate datalayer. This allows you to keep the display layer pure and easy to test by changing the props alone.

File: components/BackgroundView/BackgroundView.jsx (hunk 1 above)
Comment (thirsty-wing) [reply components/BackgroundView/BackgroundView.jsx:25]: Did as much as possible.

**Pull request merged**